[2025-06-12 13:43:00,759] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: testando_spark_submit.run_bronze_script manual__2025-06-12T16:42:30.472057+00:00 [queued]>
[2025-06-12 13:43:00,791] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: testando_spark_submit.run_bronze_script manual__2025-06-12T16:42:30.472057+00:00 [queued]>
[2025-06-12 13:43:00,792] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-06-12 13:43:00,793] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-06-12 13:43:00,794] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-06-12 13:43:00,871] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): run_bronze_script> on 2025-06-12 16:42:30.472057+00:00
[2025-06-12 13:43:00,879] {standard_task_runner.py:52} INFO - Started process 20705 to run task
[2025-06-12 13:43:00,887] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'testando_spark_submit', 'run_bronze_script', 'manual__2025-06-12T16:42:30.472057+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/teste.py', '--cfg-path', '/tmp/tmpim_gm87c', '--error-file', '/tmp/tmp5283y6ma']
[2025-06-12 13:43:00,889] {standard_task_runner.py:80} INFO - Job 16: Subtask run_bronze_script
[2025-06-12 13:43:01,124] {task_command.py:370} INFO - Running <TaskInstance: testando_spark_submit.run_bronze_script manual__2025-06-12T16:42:30.472057+00:00 [running]> on host YanArcanjo.localdomain
[2025-06-12 13:43:01,360] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=seu_email@exemplo.com
AIRFLOW_CTX_DAG_OWNER=seu_nome
AIRFLOW_CTX_DAG_ID=testando_spark_submit
AIRFLOW_CTX_TASK_ID=run_bronze_script
AIRFLOW_CTX_EXECUTION_DATE=2025-06-12T16:42:30.472057+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-06-12T16:42:30.472057+00:00
[2025-06-12 13:43:01,363] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-06-12 13:43:01,366] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'spark-submit --packages io.delta:delta-core_2.12:2.4.0             --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension"             --conf "spark.sql.catalog.spark_catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog"             --conf "spark.driver.extraJavaOptions=-Dlog4j.configurationFile=src/jobs/log4j2.properties"             ./mnt/c/Users/arcan/Documents/airflow/src/jobs/meuscript.py tabela_airflow']
[2025-06-12 13:43:01,383] {subprocess.py:85} INFO - Output:
[2025-06-12 13:43:09,254] {subprocess.py:92} INFO - ERROR StatusLogger Reconfiguration failed: No configuration found for '53bd815b' at 'null' in 'null'
[2025-06-12 13:43:09,284] {subprocess.py:92} INFO - ERROR StatusLogger Reconfiguration failed: No configuration found for 'Default' at 'null' in 'null'
[2025-06-12 13:43:10,442] {subprocess.py:92} INFO - 25/06/12 13:43:10 WARN Utils: Your hostname, YanArcanjo resolves to a loopback address: 127.0.1.1; using 10.255.255.254 instead (on interface lo)
[2025-06-12 13:43:10,457] {subprocess.py:92} INFO - 25/06/12 13:43:10 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2025-06-12 13:43:11,550] {subprocess.py:92} INFO - :: loading settings :: url = jar:file:/mnt/c/Users/arcan/Documents/airflow/.venv/lib/python3.10/site-packages/pyspark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-06-12 13:43:11,836] {subprocess.py:92} INFO - Ivy Default Cache set to: /home/yanarcanjo/.ivy2/cache
[2025-06-12 13:43:11,837] {subprocess.py:92} INFO - The jars for the packages stored in: /home/yanarcanjo/.ivy2/jars
[2025-06-12 13:43:11,866] {subprocess.py:92} INFO - io.delta#delta-core_2.12 added as a dependency
[2025-06-12 13:43:11,871] {subprocess.py:92} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-d26ab732-d2fa-4fb7-9953-b449d428b1f3;1.0
[2025-06-12 13:43:11,873] {subprocess.py:92} INFO - 	confs: [default]
[2025-06-12 13:43:12,230] {subprocess.py:92} INFO - 	found io.delta#delta-core_2.12;2.4.0 in central
[2025-06-12 13:43:12,315] {subprocess.py:92} INFO - 	found io.delta#delta-storage;2.4.0 in central
[2025-06-12 13:43:12,374] {subprocess.py:92} INFO - 	found org.antlr#antlr4-runtime;4.9.3 in central
[2025-06-12 13:43:12,433] {subprocess.py:92} INFO - :: resolution report :: resolve 544ms :: artifacts dl 17ms
[2025-06-12 13:43:12,434] {subprocess.py:92} INFO - 	:: modules in use:
[2025-06-12 13:43:12,435] {subprocess.py:92} INFO - 	io.delta#delta-core_2.12;2.4.0 from central in [default]
[2025-06-12 13:43:12,435] {subprocess.py:92} INFO - 	io.delta#delta-storage;2.4.0 from central in [default]
[2025-06-12 13:43:12,436] {subprocess.py:92} INFO - 	org.antlr#antlr4-runtime;4.9.3 from central in [default]
[2025-06-12 13:43:12,437] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-06-12 13:43:12,438] {subprocess.py:92} INFO - 	|                  |            modules            ||   artifacts   |
[2025-06-12 13:43:12,439] {subprocess.py:92} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-06-12 13:43:12,440] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-06-12 13:43:12,440] {subprocess.py:92} INFO - 	|      default     |   3   |   0   |   0   |   0   ||   3   |   0   |
[2025-06-12 13:43:12,441] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-06-12 13:43:12,446] {subprocess.py:92} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-d26ab732-d2fa-4fb7-9953-b449d428b1f3
[2025-06-12 13:43:12,447] {subprocess.py:92} INFO - 	confs: [default]
[2025-06-12 13:43:12,462] {subprocess.py:92} INFO - 	0 artifacts copied, 3 already retrieved (0kB/16ms)
[2025-06-12 13:43:13,422] {subprocess.py:92} INFO - 25/06/12 13:43:13 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-06-12 13:43:14,574] {subprocess.py:92} INFO - python3: can't open file '/tmp/airflowtmpbu9fn7g8/mnt/c/Users/arcan/Documents/airflow/src/jobs/meuscript.py': [Errno 2] No such file or directory
[2025-06-12 13:43:14,603] {subprocess.py:92} INFO - 25/06/12 13:43:14 INFO ShutdownHookManager: Shutdown hook called
[2025-06-12 13:43:14,605] {subprocess.py:92} INFO - 25/06/12 13:43:14 INFO ShutdownHookManager: Deleting directory /tmp/spark-bc863f05-06cf-4ca8-bd2c-25e559f7fe16
[2025-06-12 13:43:14,830] {subprocess.py:96} INFO - Command exited with return code 2
[2025-06-12 13:43:14,869] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/operators/bash.py", line 194, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-06-12 13:43:14,878] {taskinstance.py:1395} INFO - Marking task as UP_FOR_RETRY. dag_id=testando_spark_submit, task_id=run_bronze_script, execution_date=20250612T164230, start_date=20250612T164300, end_date=20250612T164314
[2025-06-12 13:43:14,945] {standard_task_runner.py:92} ERROR - Failed to execute job 16 for task run_bronze_script (Bash command failed. The command returned a non-zero exit code 2.; 20705)
[2025-06-12 13:43:14,976] {local_task_job.py:156} INFO - Task exited with return code 1
[2025-06-12 13:43:15,040] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
